File: src/test/samples/sample.kt
fun main() {
    // #region FirstRegion
    val x = 42
    //#endregion

    //#region SecondRegion  
    class MyClass {
        // #region   InnerRegion   
        fun myMethod() {}
        //  #endregion   ends InnerRegion 

        // #region
        fun myMethod2() {}
               //#endregion 
    }
    // #endregion
}

File: src/test/samples/sample.rs
fn main() {
    // #region FirstRegion
    let x = 42;
    //#endregion

    //    #region SecondRegion
    struct MyClass;

    impl MyClass {
        // #region InnerRegion
        fn my_method(&self) {}
        //   #endregion   ends InnerRegion

        // #region
        fn my_method2(&self) {}
        //#endregion
    }
    //  #endregion
}
